uint16 lift_cmd_msg 
uint16 ir_cmd_msg 
uint16 ir_dist_left_msg
uint16 ir_dist_right_msg 
uint32 angle_msg
bool mount_status_msg
bool alignement_msg